core/sqf/sqenvcom.sh

```diff
@@ -795,13 +799,9 @@ fi
 
 # Non-standard or newer version tools
 export BISON="${TOOLSDIR}/bison_3_linux/bin/bison"     # Need 1.3 version or later
-
-
 export LLVM="${TOOLSDIR}/dest-llvm-3.2"
 export UDIS86="${TOOLSDIR}/udis86-1.7.2"
 export ICU="${TOOLSDIR}/icu4.4"
-export QT_TOOLKIT="${TOOLSDIR}/Qt-4.8.5-64"
-
 
```

Comment:
Is there a requirement to build the QT_TOOLKIT?  I asked around and it seemed to be an optional product. This would be one less dependency that the user has to provide.  Given that I was unable to get QT to build on a VM, (it kept wanting more and more dependencies which I could not find).   I assumed that if someone wants this tool, then we can provide instructions on how to build it.  I can add a section on the build page to discuss how to build and use this tool.